src/containers/sb-file-uploader.jsx

```diff
@@ -146,8 +149,10 @@ class SBFileUploader extends React.Component {
                 });
         }
     }
-    handleClick () {
+    handleClick (e) {
         // open filesystem browsing window
+        e.stopPropagation();
+        this.props.openFileInput();
```

Comment:
One issue: having set this state property for the menus reducer, we don't reliably _unset_ it. E.g., if you cancel the file upload when the file dialog is open, the file menu stays open and won't close on regular mouseclicks.

Reply:
There is no way to catch cancel event :(